content/docs/user-guide/scripting/script-canvas/editor-reference/text-replacement.md

```diff
@@ -0,0 +1,499 @@
+---
+linkTitle: Text Replacement
+title: Script Canvas Node Text Replacement
+description: Learn how to customize the text on Script Canvas nodes in Open 3D Engine (O3DE).
+weight: 500
+---
+
+# Script Canvas - Text Replacement User Documentation
+
+Script Canvas' text replacement system allows all available nodes to be further customized. Method names, categories, argument names, tooltips can all be updated to improve their look once in a Script Canvas graph.
+
+## Rules for updating text
+
+The .names files have certain fields that should not be modified, they are used by the system to create a key into a database of text fields. 
+
+**The following fields should NOT be changed**:
+
+`base`
+
+`context`
+
+`variant`
+
+`typeid`
+
+Any changes to these fields will result in the Script Canvas editor being unable to find the desired text to update and will result in the translation data being ignored.
+
+The data within the "details" field can be updated with the following fields:
+
+`name` The name of the node, argument or return type
+
+`category` The category to place the node in, supports nested categories by using the / character (i.e. "Gameplay/Missions")
+
+`tooltip` Provides a tooltip to the element, this can be the node itself, or any of the node slots when the tooltip is added on a method or arguments
+
+`subtitle` This can be used on the entry details object, it allows overriding the node's subtitle
+
+
+## Getting started
+
+### Example: Generating a translation file to replace text
+
+Open the Script Canvas editor
+Navigate or search for the desired node in the Node Palette
+
+We will use AWSGameLiftPlayer in this example
+
+ ![Image 0](/images/user-guide/scripting/script-canvas/text-replacement-0.png)
+
+Right click on any of the nodes under AWSGameListPlayer
+
+![Image 1](/images/user-guide/scripting/script-canvas/text-replacement-1.png)
+
+And select "Generate Translation"
+
+This will open a file explorer window to an auto-generated file called: *AWSGameLiftPlayer.names*
+
+If you do not wish to make any changes to the auto-generated data, you can close the Script Canvas editor window and reopen it and you will see some changes have taken effect.
+
+![Image 2](/images/user-guide/scripting/script-canvas/text-replacement-2.png)
+
+All the names are now separated by a space and the naming follows the Script Canvas naming convention to use Camel Case.
+
+### Changing the node category
+
+We have succesfully generated translation data for a node, but its category is still under "Other". To change this we will now edit the generated file *AWSGameLiftPlayer.names*
+
```

Comment:
They might have already closed the .names file. Instruct them how to open it again, in case they have closed it. (Using **Explore Translation Data**, I think?)